core/src/main/resources/hudson/util/HudsonIsRestarting/index_da.properties

```diff
@@ -20,5 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Your\ browser\ will\ reload\ automatically\ when\ Jenkins\ is\ ready.=Din browser vil genindlæse siden automatisk når Jenkins er klar.
+Your\ browser\ will\ reload\ automatically\ when\ Jenkins\ is\ ready=Din browser vil genindlæse siden automatisk når Jenkins er klar.
 Please\ wait\ while\ Jenkins\ is\ restarting=Vent venligst imens Jenkins genstarter
```

Comment:
there's still a number of translations outstanding for this one `Please\ wait\ while\ Jenkins\ is\ restarting`

Either update or remove translation please